Actions/WorkflowInitialize/WorkflowInitialize.ps1

```diff
@@ -9,18 +9,36 @@ try {
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
     . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-TestRepoHelper.ps1" -Resolve)
 
-    $ap = "$ENV:GITHUB_ACTION_PATH".Split('\')
+    # $ENV:GITHUB_ACTION_PATH is
+    #                                      -4       -3         -2       -1
+    # linux:            /home/runner/work/owner/AL-Go-Actions/branch/WorkflowInitialize
+    # windows:  C:\Users\runneradmin\work\owner\AL-Go-Actions\branch\WorkflowInitialize
```

Comment:
For me the action path isn't consistent with / and \\. I'm getting this action path on a windows runner 🤷‍♂️  

`D:\a\_actions\aholstrup1\AL-Go\branch\Actions/WorkflowInitialize`

Maybe we need to replace all \\ with / or vice versa? To ensure it's consistent

Reply:
Will just split('/\') - that should take care of this.